src/swdmt/redistricting/VoterTest.java

```diff
@@ -23,19 +31,29 @@ public void affiliationAttributeNullTest() {
         assertThat(voter2.affiliation(), is(Party.NONE));
     }
 
-    @Test(expected = IllegalArgumentException.class)
-    public void invalidLocationTest() {
-        Voter voter1 = new Voter(Party.NONE, null);
+    /**
+    * When passed a null location should throw an illegal argument exception
+    */
+    @Test
+    public void invalidLocationTest() throws IllegalArgumentException {
+        assertThrows(IllegalArgumentException.class, () -> {
+            Voter voter1 = new Voter(Party.NONE, null);
+        });
     }
 
+    /**
+    * When passed various location attributes they match expected values
+    */
     @Test
     public void locationAttributeTest() {
         Voter voter1 = new Voter(Party.NONE, new Location(0, 0));
-        assertThat(voter1.location(), is(new Location(0, 0)));
```

Comment:
Why was the actual ordering of construction and testing changed?

Note that this change of ordering can result in the test (original line 34) not being executed if the invocation of the constructor (original lines 35-37) fails.

This appears to be a potential behavioral change to the test rather than just a refactoring to address upgrading from JUnit 4 to JUnit 5.